### ISSUE TITLE
Be able to use nuget gem in a simple way

### ISSUE THREAD

**Issue by wallymathieu** [Albacore/albacore#181]
It would be nice if albacore could use the nuget-exe from the nuget gem if it is available
https://rubygems.org/gems/nuget/
so that you dont have manually specify the nuget exe path

Perhaps by using something like

```
nugets_pack :create_nugets do |p|
  ...
 p.use_nuget_gem_exe
  ....
end
```

?

**Comment by haf** [comment]
See also https://github.com/Albacore/albacore/issues/174 and https://github.com/Albacore/albacore/commits/feature/paket-pack

**Comment by wallymathieu** [comment]
I've added a pull request with the code: #182

**Comment by haf** [comment]
Great! If you write `closes #181` in one of the commits for that PR, this issue will be closed when we merge your PR.

**Comment by wallymathieu** [comment]
Forgot to write "closes ..."
